SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
```

Comment:
What do these version things stand for in context of our app or is this just a template? 🤔

Reply:
Hi @NiallJoeMaher. It means the versions of the project that has been released. Think of it as Apple release a new version of their iOS products.